package/Dockerfile

```diff
@@ -53,19 +68,40 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 101.0.0+up0.3.3
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=101.0.1+up0.5.1
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=2.0.0+up0.3.0
-ENV CATTLE_CSP_ADAPTER_MIN_VERSION=2.0.0
+ENV CATTLE_FLEET_MIN_VERSION=100.0.5+up0.3.11
```

Comment:
These versions are older than what's in the upstream? Is that intentional? IIRC we make these min versions match the latest in the bundled charts so that we guarantee we get the version we want during an upgrade. That info is probably missing from the BFS instructions though.

Reply:
I am not sure how these got changed, but I verified the latest ones on the docker image are the values in the upstream.  I am putting those values back to match the upstream.